### ISSUE TITLE
[Feature] Start generating .webm for Markers immediately after marker creation

### ISSUE THREAD

**Issue by ALonelyJuicebox** [stashapp/stash#4067]
In its current state, Stash requires that a user manually run a generate task on the entire db, or manually start a generate task specifically on that scene in order to generate the animated webm for a given marker. 

This two step process isn't terribly intuitive.
I propose that Stash just start generation of a marker as soon as the user defines where the marker is located.

There's an existing GQL mutation (`sceneMarkerCreate`) that could be modified to have an option to specify whether or not the webm should be generated. Setting this value to default to "false" would help avoid any issues with existing plugins.

Given the direct user impact, I would prefer this feature be added as a core functionality rather than a plugin.
Link to a short relevant RFC on discord here (https://discord.com/channels/559159668438728723/559399634318131210/1142009881847676959)

Thanks! 🙂

**Comment by scruffynerf** [comment]
I agree adding a Do It Now option makes sense, but adding a hook allows us more flexibility to do it on demand, batch it up, mleave it manual, etc.

Make it a setting (generate markers at once), but also add a hook for when the mutation is invoked.

**Comment by feederbox826** [comment]
As a stopgap solution, I created a plugin https://github.com/feederbox826/plugins/tree/main/plugins/markergen

**Comment by discourse-stashapp** [comment]
This issue has been mentioned on **Stash**. There might be relevant details there:

https://discourse.stashapp.cc/t/how-to-generate-webms-from-markers/4681/1
